babi/main.py

```diff
@@ -133,7 +134,11 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
 
     if '-' in args.filenames:
         print('reading stdin...', file=sys.stderr)
-        stdin = sys.stdin.read()
+        stdin_bytes = sys.stdin.buffer.read()
```

Comment:
you can write this as `sys.stdin.buffer.read().decode()` -- the default encoding in python3 is UTF-8